content/en/docs/concepts/scheduling-eviction/scheduling-framework.md

```diff
@@ -75,6 +75,14 @@ node, the scheduler will call filter plugins in their configured order. If any
 filter plugin marks the node as infeasible, the remaining plugins will not be
 called for that node. Nodes may be evaluated concurrently.
 
+### PostFilter {#post-filter}
+
+These plugins are called after Filter phase, but only when no feasible nodes
```

Comment:
These plugins run after the Filter phase but only if there are no feasible nodes available for scheduling.